Src/FCConverter.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef GS_FAVORITE_COVNERTER_CONVERTER_HPP
+#define GS_FAVORITE_COVNERTER_CONVERTER_HPP
+
+#include "Folder.hpp"
+
+namespace FC {
+
+class Converter {
+public:
+	Converter (const GS::UniString& sourceFolderPath,
+               const GS::UniString& destinationFolderPath);
```

Comment:
This can be a one-liner
```suggestion
	Converter (const GS::UniString& sourceFolderPath, const GS::UniString& destinationFolderPath);
```

Reply:
Uh the suggestion is a little bit confusing 😄